Flow.Launcher/Helper/FileExplorerHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.IO;
+
+namespace Flow.Launcher.Helper
+{
+    public class FileExplorerHelper
+    {
+
+        /// <summary>
+        /// Gets the path of the file explorer that is currently in the foreground
+        /// </summary>
+        public static string GetActiveExplorerPath()
+        {
+            var explorerWindow = GetActiveExplorer();
+            string locationUrl = explorerWindow.LocationURL;
+            if (!string.IsNullOrEmpty(locationUrl))
+            {
+                return new Uri(locationUrl).LocalPath;
```

Comment:
This ended up being the most reliable way of getting the explorer path. I think edge cases are folders like the desktop, recycle bin, documents folder, etc.